packages/v4/src/content/design-guidelines/styles/typography/typography.md

```diff
@@ -22,36 +22,74 @@ We use Red Hat Display and Red Hat Text. Red Hat Display should be used for **la
+  title="Default body text"
+  note="This is the default text style for paragraphs, lists, tables, etc. Some components use RedHatText at 700 font weight, which is RedHatText Medium (such as alerts and navigation)"
   styleProps = {styleProps.body} />
 <TitleLevel asGrid title="Small text" styleProps = {styleProps.small} />
-<TitleLevel 
-  asGrid 
-  title="Tiny text*"
-  note="*Not to be used in content blocks (only used with data visualizations when 14px is not small enough)"
+<TitleLevel
+  asGrid
+  title="Tiny text"
   styleProps = {styleProps.tiny} />
 <TitleLevel
   className="ws-typography-code-font"
-  asGrid 
-  title="Code*"
-  note="*Used for code blocks "
+  asGrid
+  title="Code"
+  note="Used for code blocks "
   styleProps = {styleProps.code} />
 
-## Line height
-<div style={{marginBottom: '32px'}}>When measuring spacing in text-based content, plan for line height.
+  ### Customizing heading levels
+  If you use the [title component](/components/title) to create headings on your page, you can modify the default relationship between heading levels and text sizes. In using title, you must specify a heading level that will apply to the target text. The table below shows the default mapping of heading level to text size:
+
+
+  | Heading level | Default size|
+  |----|-----------|
+  | h1 | 2xl (24px)|
+  | h2 | xl (20px) |
+  | h3 | lg (18px) |
+  | h4 | md (16px) |
+  | h5 | md (16px) |
+  | h6 | md (16px) |
+
+
+  Using this component, you can customize the visual hierarchy of text on your page while keeping the semantic hierarchy consistent with expectations for accessibility. For example, you may decide that 20px secondary headings are too large and you want to drop the size of these headings from 20px (xl) to 18px (lg). Rather than make your secondary headings h3’s, you should use the title component to keep them as h2 headings but change the associated text size from `xl` to `lg`.  You can also choose to use larger primary headers by changing the default text size as the title component will support text sizes up to 4xl (36px).
+
+  No matter what visual customizations you choose to make, you should always maintain good visual hierarchy and mapping between heading levels and text sizes. In most cases, h1 should always be your largest heading and subheadings should get progressively smaller as you move down the hierarchy. Exceptions to this rule do occur. For example, there might be cases where card titles will use a text size that is larger than the h1 page title. We advise this approach to be used sparingly and only when it may be important to highlight critical data as in a dashboard.
+
+## Line height and spacing
+<div style={{marginBottom: '32px'}}>Spacing represents margins above and below text. When measuring spacing in text-based content, plan for line height. Line height should be 1.5 times larger than text size. Use spacers to provide appropriate spacing between lines of text.
+
```

Comment:
there is no space between these paragraphs, I'm assuming because of the `<div>` - the text here isn't being added to a paragraph. You may try adding a new line or two so that the "Spacing represents..." starts on its own line? Maybe that will trigger that text to be put in a paragraph by markdown.

Reply:
Just removed the <div>. Not sure why it was there in the first place.